src/client/VZSettings.tsx

```diff
@@ -203,45 +174,36 @@ export const VZSettings = ({
           <select
             className="form-select"
             value={theme}
-            onChange={handleThemeChange}
+            onChange={(e) => setTheme(e.target.value as ThemeLabel)}
```

Comment:
The `useCalback` is a good optimization to keep in place. Otherwise performance may degrade if we do this all over the place (since removing and re-adding event listeners each render can be expensive of there are lots of these).